files/fr/web/javascript/reference/operators/logical_and/index.md

```diff
@@ -126,7 +126,7 @@ bCondition1 || (bCondition2 && bCondition3);
 sera toujours égale à :
 
 ```js
```

Comment:
You can add `-nolint` prefix to prevent linting, see:

- https://github.com/mdn/content/blob/694fcee3661206baded9795045cab396836fac60/files/en-us/web/javascript/reference/operators/logical_and/index.md?plain=1#L151C1-L151C13
- https://developer.mozilla.org/fr/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#suppression_du_formatage

````suggestion
```js-nolint
````

Reply:
Thanks! Didn't think about that. 🙏